Fred2/test/external/TestExternalCleavagePrediction.py

```diff
@@ -13,8 +13,7 @@ class TestExternalCleavagePredictonClass(unittest.TestCase):
 
     def setUp(self):
         self.seqs = [Peptide("SYFPEISYFP"),
-                     Protein("IHTIEPFYSIHTIEPFYSIHTIEPFYSIHTIEPFYSIHTIEPFYS", transcript_id="ID-01", gene_id="FOXP3"),
-                     Transcript("")]
+                     Protein("IHTIEPFYSIHTIEPFYSIHTIEPFYSIHTIEPFYSIHTIEPFYS", transcript_id="ID-01", gene_id="FOXP3")]
```

Comment:
One test was throwing errors because it was calling all seas and I had added a transcript to fix the raise value error